### PR TITLE
Forge is NeoForge now (Port to 1.20.4)

### PR DESCRIPTION
**Pull request by dima-dencep**
Don't forget to change the branch 😼

### CODE REVIEW

File: forge/src/main/resources/META-INF/mods.toml
Comment (KosmX) [inline forge/src/main/resources/META-INF/mods.toml:1]: Thanks for the proper neoforge mods.toml

**Comment by KosmX** [comment]
Everything looks good, gradle wrapper checksum verified, merging after some tests.
Thank you!

File: fabric/build.gradle [hunk 1]
```diff
@@ -27,20 +27,20 @@ repositories {
 
 dependencies {
     modImplementation "net.fabricmc:fabric-loader:${rootProject.loader_version}"
-    //modApi "net.fabricmc.fabric-api:fabric-api:${rootProject.fabric_version}"
```
Comment (KosmX) [inline fabric/build.gradle:30]: This was not necessary

Comment (dima-dencep) [reply fabric/build.gradle:30]: ?

Comment (KosmX) [reply fabric/build.gradle:30]: bendy-lib does not depend on Fabric API but you did uncomment this line.

**Pull request merged**